pkg/api/peer_req_resp.go

```diff
@@ -44,3 +48,25 @@ Example:
 Note - Cannot use query parameters if peerid is also supplied.
 */
 type PeerListResp []PeerGetResp
+
+// GetMetadataSize returns the size of the peer metadata in PeerAddReq
+func (p PeerAddReq) GetMetadataSize() int {
```

Comment:
`PeerAddReq` should be a pointer handle - `*PeerAddReq`

Same for `PeerEditReq`